src/qibocal/auto/runcard.py

```diff
@@ -38,6 +39,8 @@ class Runcard:
     """Structure of an execution runcard."""
 
     actions: List[Action]
+    qubits: Optional[List[int]] = Field(default_factory=list)
```

Comment:
I tested this branch for the 25q chip with QM and the string qubit issue is fixed just by changing this line to
```suggestion
    qubits: Optional[List[Union[int, str]]] = Field(default_factory=list)
```

So far I tested only the resonator spectroscopy and worked after making only this change and merging qiboteam/qibolab#341 to qiboteam/qibolab#349. I will try to test some more routines later today and report if I find any issues.

Reply:
That's great!
Thanks @stavros11. Let us know if see other issues.